ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/port-channel-interfaces.j2

```diff
@@ -243,6 +243,9 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{# IP NAT Documentation #}
```

Comment:
I would prefer to have all the IP NAT interface documentation in a single section covering all interfaces. Together with the global nat configuration. This way there is one place to look for IP NAT stuff. We could avoid these extra includes and just parse the different interface types one-by-one in there, since they all share the same data model.

Reply:
@carlbuchmann @gmuloc opinions?

Reply:
I had the same thoughts during working on the code. I decided to keep NAT documentation near to the interface IPv4 documentation. But anyway, there are pros and cons for both ways. I will follow the decision of the maintainers.

Reply:
In interest of time we will keep it as is for now. Thank you for your patience.